index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" rel="stylesheet">
+    <link href="style.css" rel="stylesheet">
+    <title>Youtube</title>
+</head>
+<body>
+    <nav class="navbar navbar-light bg-light">
+        <div class="container-fluid">
+          <a class="navbar-brand " href="#">
+            <img class="logo"  src="images/youtube_logo.PNG" alt="" width="110" height="51" class="d-inline-block align-text-top">
+          </a>
+        </div>
+      </nav>
+
+    <div class="container">
+        <div class="row">
+          <div class="col-sm-8">
+                <div class="embed-responsive embed-responsive-21by9">
+                    <iframe width="737" height="415" src="https://www.youtube.com/embed/mGNUaZekwMs" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+              </div>
+              <h1 class="video-caption mt-3">Jake Paul vs. Anderson Silva second Presser: "I followed you back on Instagram! Follow me back!</h1>
+        </div>
+          <div class="col-sm-4">
+            <div class="row mt-3">
+                <div class="col-sm-6">
+                    <a href="https://www.youtube.com/watch?v=TNwnyA_N9ZM&t=1264s">
+                    <img src="images/image-2.png" class="img-fluid" alt="">
```

Comment:
- [ ] Add alt text to all your images😣

Reply:
Nice work explaining the improvements to the student. However, Instead of giving orders, you could make suggestions.
You could say for example :
Well done adding attributes 👍, however, I think you have forgotten to complete `alt`  attribute for semantics reasons Kindly fix it 🤝.
What do you think about refreshing yourself about [professional communication in a code review?](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/code-review/articles/give_code_review_basics.md#how-to-communicate) :🤗